holoclean.py

```diff
@@ -239,10 +241,13 @@ def load_data(self, name, fpath, na_values=None,
             specifies the column containing the source for each "mention" of an
             entity.
         """
-        status, load_time = self.ds.load_data(name, fpath, na_values=na_values,
-                entity_col=entity_col, src_col=src_col)
+        status, load_time = self.ds.load_data(name,
+                                              fpath,
+                                              na_values=na_values,
+                                              entity_col=entity_col,
+                                              src_col=src_col)
         logging.info(status)
-        logging.debug('Time to load dataset: %.2f secs'%load_time)
+        logging.debug('Time to load dataset: %.2f secs' % load_time)
```

Comment:
we can use a comma here instead of `%` since `logging` handles string interpolation.

Reply:
ok